src/main/java/Control/Scenario/UC_FlatSummary.java

```diff
@@ -70,7 +70,7 @@ public Flat getFlatSummary(@NotNull final Integer flatId) {
         // ATTEMPT to eliminate WEBSERVLET only falling asleep of connections
         db.beforeSqlExecution(false);
 
-        T_Flat t_flat = getFlatById(flatId);
+        T_Flat t_flat = getTFlatById(flatId);
```

Comment:
co je getTFlat miesto getFlat? akoze table je to Tcko?

Reply:
Hej, lebo jednoducho tam bol konflikt, kedže vraciame T_Flat a kedze ludo tam ma tie svoje vlastne modely Flat tak getFlat bol problemovy